src/overmap.cpp

```diff
@@ -2507,6 +2512,9 @@ tripoint overmap::draw_overmap(const tripoint &orig, const draw_data_t &data)
             } else {
                 show_explored = true;
             }
+        } else if (action == "TOGGLE_CURSOR") {
+            uistate.overmap_cursor = !uistate.overmap_cursor;
+
```

Comment:
Minor: That blank line isn't needed. Besides that, the code looks good, but I'll compile and test it to be sure.

Also, best practice is to keep your master branch up to date with CleverRaven's master branch, and to do PRs on separate feature branches (see the Guidelines section in `CONTRIBUTING.md`).